cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/GetLogic.java

```diff
@@ -35,26 +37,27 @@
      * @param spaceDevelopersOperations SpaceDevelopersOperations
      * @param servicesOperations ServicesOperations
      * @param applicationsOperations ApplicationsOperations
-     * @param clientOperations ClientOperations
+     * @param propertyUtils PropertyUtils
      * @param loginOptions LoginCommandOptions
      * @return ConfigBean
      * @throws GetException if an error occurs during the information retrieving
      */
     public ConfigBean getAll(SpaceDevelopersOperations spaceDevelopersOperations,
                              ServicesOperations servicesOperations,
                              ApplicationsOperations applicationsOperations,
-                             ClientOperations clientOperations,
+                             PropertyUtils propertyUtils,
```

Comment:
I don't think we need DI here.

Reply:
If not I can not mock it there.

Reply:
I see. Please see https://github.com/cf-control/cf-control/pull/148#discussion_r457273370.